src/main/java/com/puppycrawl/tools/checkstyle/filters/JavaPatchFilterElement.java

```diff
@@ -43,12 +45,55 @@
      */
     private static final String MAX = "max";
 
+    /**
+     * Mapping between a check and its ancestor token types.
+     */
+    private static final Map<String, List<Integer>> CHECK_TO_ANCESTOR_NODES_MAP = new HashMap<>();
+
+    static {
+        CHECK_TO_ANCESTOR_NODES_MAP.put("ArrayTrailingComma",
+                Arrays.asList(TokenTypes.VARIABLE_DEF, TokenTypes.ARRAY_INIT));
```

Comment:
why do we need VARIABLE_DEF? Isn't ARRAY_INIT enough?

Reply:
removed, it is enough.